feat(JGU-91): added User Preferences documentation (#30)
mia-platform/documentation: docs/development_suite/user-settings/user-preferences.md

---
id: user-preferences
title: User Preferences
sidebar_label: User Preferences
---

### Manage your Console Preferences

> Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the console for your account. 

Reviewer: ```suggestion
Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the Console for your account. 
```

> Hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some behaviors of the Console for your account. 

Reviewer 2: ```suggestion
After hovering over the user profile picture, from the dropdown menu, click on the `User Preferences` item to view and manage some preferences in Console for your account. 
```

![user preferences dropdown](./img/user-preferences-dropdown.png)

The preferences that you can manage are:

> - **Company invitation email**: this setting manages whether or not you receive an email notification when you are added on a new Company.

Reviewer 2: ```suggestion
- **Company invitation email**: through this setting you can decide whether or not to receive an invitation email when you are added on a Company.
```
